v2.0/backup.md

```diff
@@ -141,36 +151,42 @@ Per our guidance in the [Performance](#performance) section, we recommend starti
 ### Backup a Single Table or View
 
 ~~~ sql
-> BACKUP bank.customers TO 'azure://acme-co-backup/table-customer-2017-03-27-full?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co'
-AS OF SYSTEM TIME '2017-06-09 16:13:55.571516+00:00';
+> BACKUP bank.customers to 'gs://acme-co-backup/database-bank-2017-03-27-weekly' AS OF SYSTEM TIME '2017-03-26 23:59:00';
 ~~~
 
 ### Backup Multiple Tables
 
 ~~~ sql
-> BACKUP bank.customers, bank.accounts TO 'azure://acme-co-backup/tables-accounts-customers-2017-03-27-full?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co'
-AS OF SYSTEM TIME '2017-06-09 16:13:55.571516+00:00');
+> BACKUP bank.customers, bank.accounts to 'gs://acme-co-backup/database-bank-2017-03-27-weekly' AS OF SYSTEM TIME '2017-03-26 23:59:00'; 
```

Comment:
Same as above:

It'd be a nice UX improvement to wrap lines in these code samples:

```
> BACKUP bank.customers, bank.accounts \
TO 'gs://acme-co-backup/database-bank-2017-03-27-weekly' \
AS OF SYSTEM TIME '2017-03-26 23:59:00'; 
```